solidity/test/BondedECDSAKeepFactoryTest.js

```diff
@@ -307,44 +309,39 @@ contract("BondedECDSAKeepFactory", async (accounts) => {
       )
     })
 
-    it("returns false if the operator stake is below minimum", async () => {
+    it("returns false if the operator stake dropped", async () => {
+      await stakeOperators(members, minimumStake.muln(50))
+
       await keepFactory.registerMemberCandidate(application, {
         from: members[0],
       })
-
-      await stakeOperators(members, minimumStake.subn(1))
+      await stakeOperators(members, minimumStake.muln(49))
 
       assert.isFalse(
         await keepFactory.isOperatorUpToDate(members[0], application)
       )
     })
 
-    it("returns true if the operator stake changed insufficiently", async () => {
+    it("returns false if the operator stake dropped below minimum", async () => {
       await keepFactory.registerMemberCandidate(application, {
         from: members[0],
       })
 
-      // We multiply minimumStake as sortition pools expect multiplies of the
-      // minimum stake to calculate stakers weight for eligibility.
-      // We subtract 1 to get the same staking weight which is calculated as
-      // `weight = floor(stakingBalance / minimumStake)`.
-      await stakeOperators(members, minimumStake.mul(new BN(2)).sub(new BN(1)))
+      await stakeOperators(members, minimumStake.subn(1))
 
-      assert.isTrue(
+      assert.isFalse(
         await keepFactory.isOperatorUpToDate(members[0], application)
       )
     })
 
-    it("returns false if the operator stake is above minimum", async () => {
+    it("returns false if the operator stake increased", async () => {
```

Comment:
```suggestion
    it("returns true if the operator stake increased", async () => {
```
?

Reply:
https://github.com/keep-network/keep-ecdsa/pull/377/commits/7b311d767ff61f09f1853fec5dcfb012230efab7

Reply:
Nope, reworked in https://github.com/keep-network/keep-ecdsa/pull/377/commits/3a61bf47809ab3b8fa01c27fc79473fa80934464